applications/ipc_radio/README.rst

```diff
@@ -0,0 +1,125 @@
+.. _ipc_radio:
+
+IPC radio firmware
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The IPC radio firmware allows to use the radio peripheral from another core in a multicore device.
+
+Application overview
+********************
+
+You can use this firmware as a general serialized radio peripheral.
+
+The firmware supports both Bluetooth® Low Energy and IEEE 802.15.4 simultaneously.
+In addition, you can configure the Bluetooth selection as an HCI :ref:`bluetooth_controller` or an RPC serialization interface (:ref:`ble_rpc`).
+
+HCI IPC serialization
+=====================
+
+The firmware exposes the :ref:`bluetooth_controller` support to another core using the IPC subsystem.
+
+Host for nRF RPC Bluetooth Low Energy
+=====================================
+
+The firmware is running the full Bluetooth Low Energy stack.
+It receives serialized function calls that it decodes and executes, then sends response data to the client.
+
+The serialization includes:
+
+* :ref:`bt_gap`
+* :ref:`bluetooth_connection_mgmt`
+* :ref:`bt_gatt`
+* :ref:`Bluetooth Cryptography <bt_crypto>`
+
+IEEE 802.15.4
+=============
+
+The firmware exposes radio driver support to another core using the IPC subsystem.
+
+Requirements
+************
+
+The firmware supports the following development kits:
+
+.. table-from-sample-yaml::
+
+To automatically attach the firmware image, you need to use the :ref:`sysbuild`.
```

Comment:
```suggestion
To automatically attach the firmware image, you need to use the :ref:`zephyr:sysbuild`.
```
Please add reference here to nRF Connect for Visual Studio Code, which already supports sysbuild and is our recommended IDE: https://docs.nordicsemi.com/bundle/nrf-connect-vscode/page/guides/build_overview.html#system-build-sysbuild and https://docs.nordicsemi.com/bundle/nrf-connect-vscode/page/guides/build_config.html